### ISSUE TITLE
object-curly-spacing bugs on export X from Y

### ISSUE THREAD

**Issue by Anahkiasen** [eslint/eslint#5590]
**What version of ESLint are you using?**
2.4.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint

**Please show your full configuration:**
```yaml
parser: 'babel-eslint'
rules:
    object-curly-spacing: 2
```

**What did you do? Please include the actual source code causing the issue.**
```js
export SideMenu from './SideMenu';
export SideMenuLink from './SideMenuLink';
```

**What did you expect to happen?**
Nothing

**What actually happened? Please include the actual, raw output from ESLint.**
```
/myapp/resources/assets/js/Components/Navigation/SideMenu/index.js
  1:1   error  There should be no space after 'export'  object-curly-spacing
  1:17  error  There should be no space before 'from'   object-curly-spacing
  2:1   error  There should be no space after 'export'  object-curly-spacing
  2:21  error  There should be no space before 'from'   object-curly-spacing
```

**Comment by mysticatea** [comment]
Thank you for this issue.

It's known issue of `babel-eslint`.
Please use [eslint-plugin-babel](https://github.com/babel/eslint-plugin-babel#readme).

**Comment by Anahkiasen** [comment]
My bad! Thanks for the pointer
